histoqc/SaveModule.py

```diff
@@ -36,6 +36,39 @@ def saveFinalMask(s, params):
     return
 
 
+def saveAssociatedImage(s, key:str, dim:int):
+    logging.info(f"{s['filename']} - \tsaveMacro")
+    osh = s["os_handle"]
+
+    if not key in osh.associated_images:
+        message = f"{s['filename']}- \tsave{key} Can't Read '{key}' Image from Slide's Associated Images"
+        logging.warning(message)
+        s["warnings"].append(message)
+        return
+    
+    # get asscociated image by key
+    associated_img = osh.associated_images[key]
+    (width, height)  = associated_img.size
```

Comment:
this is okay since i don't think we have a standard for histoqc, but when you look at the other tools i think you're more likely to see "nrow" and "ncol" instead of "width" and "height". just an FYI

Reply:
should **ncol** correspond to **height** and **nrow** correspond to **width**, right?

Reply:
i think its nrow = height, ncol = width?